### PR TITLE
Add trace methods benchmarks

### PR DESCRIPTION
**Pull request by rmoreliovlabs**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/Web3ConnectorE2E.java [hunk 1]
```diff
@@ -48,6 +51,7 @@ private Web3ConnectorE2E(String host) {
                 .connectTimeout(Duration.ofSeconds(120))
                 .build();
         this.web3j = new RskWeb3j(new HttpService(host, httpClient));
+        this.traceModuleWeb3j = new RskTraceModuleWeb3j(new HttpService(host));
```
[the remark this answers is not in the document]
Comment (rmoreliovlabs) [reply rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/Web3ConnectorE2E.java:54]: Done

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/RskTraceModuleWeb3j.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import java.util.stream.Collectors;
+
+public class RskTraceModuleWeb3j extends JsonRpc2_0Web3j {
+
+    public static class TraceFilterRequest {
+        private String fromBlock;
+        private String toBlock;
+        private List<String> fromAddress;
+        private List<String> toAddress;
+        private Integer after;
```
[the remark this answers is not in the document]
Comment (rmoreliovlabs) [reply rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/RskTraceModuleWeb3j.java:38]: Done

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/RskTraceModuleWeb3j.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        private Integer count = 10_000;
```
[the remark this answers is not in the document]
Comment (rmoreliovlabs) [reply rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/RskTraceModuleWeb3j.java:39]: Done

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/RskTraceModuleWeb3j.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        }
+
+        public TraceFilterRequest(String fromBlock, String toBlock, List<String> fromAddress, List<String> toAddress) {
+            this.fromBlock = fromBlock;
+            this.toBlock = toBlock;
+            this.fromAddress = fromAddress;
+            this.toAddress = toAddress;
+        }
+
+        public void setFromBlock(String fromBlock) {
```
[the remark this answers is not in the document]
Comment (rmoreliovlabs) [reply rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/RskTraceModuleWeb3j.java:56]: Done

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/Web3ConnectorE2E.java [hunk 5]
```diff
@@ -203,6 +232,16 @@ private String ethNewFilter(EthFilter filter) throws HttpRpcException {
         }
     }
 
+    private RskWeb3j.GenericJsonResponse sendRequest(Supplier<Request<?, RskWeb3j.GenericJsonResponse>> supplier) throws HttpRpcException {
```
[the remark this answers is not in the document]
Comment (rmoreliovlabs) [reply rskj-core/src/jmh/java/co/rsk/jmh/web3/e2e/Web3ConnectorE2E.java]: Done

File: rskj-core/src/jmh/resources/conf/regtest.conf [hunk 6]
```diff
@@ -44,4 +44,14 @@ debug.txFrom=0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826
 debug.block=null
 debug.transactionVT=null
 debug.transactionContractCreation=null
-debug.transactionContractCall=null
\ No newline at end of file
+debug.transactionContractCall=null
+
+############################
+# trace
+############################
```
[the remark this answers is not in the document]
Comment (Vovchyk) [reply rskj-core/src/jmh/resources/conf/regtest.conf:51]: @rmoreliovlabs , seems this is still not covered, isn't it?

**Review by Vovchyk — CHANGES_REQUESTED** [review]

File: rskj-core/src/jmh/resources/conf/regtest.conf (hunk 6 above)
[the remark this answers is not in the document]
Comment (rmoreliovlabs) [reply rskj-core/src/jmh/resources/conf/regtest.conf:51]: Addressed

File: rskj-core/src/jmh/java/co/rsk/jmh/web3/plan/GetLogsPlan.java [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+        this.ethFilterId = generateNewFilterId();
+    }
+
+    public String getEthFilterId() {
+        return ethFilterId;
+    }
+
+    private String generateNewFilterId() throws BenchmarkWeb3Exception {
+        String blockHash = configuration.getString("getLogs.blockHash");
```
Comment (asoto-iov) [inline rskj-core/src/jmh/java/co/rsk/jmh/web3/plan/GetLogsPlan.java:48]: I would extract this logic to setup stage

**Review by asoto-iov — APPROVED** [review]

**Pull request merged**